hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -40,32 +53,33 @@ def get_json_value(obj, key, default):
 #########################################
 class srt(Resource):
     def post(self, method, edge_id):
-        http_ret = 201
+        response = Response(status=201)
```

Comment:
The simpler function is `flask.make_response()` which accepts json object and http returnn code.
For example, if we need to return normal `recordId` from dbus call, then it can be simply like the below;

```
  response = make_response (jsonify(recordId=return_id), 201)
```